internal/appconfig/serde.go

```diff
@@ -15,6 +15,12 @@ import (
 	"github.com/superfly/flyctl/iostreams"
 )
 
+const flytomlHeader = `# fly.toml file generated for %s on %s\n\n", c.AppName, time.Now().Format(time.RFC3339))
+#
+# See https://fly.io/docs/reference/configuration/ for reference documentation about this file
```

Comment:
```suggestion
# See https://fly.io/docs/reference/configuration/ for  information about how to use this file.
```

Just looking for ways to connect more pathways in peoples' brains

Reply:
that is perfect. thanks